packages/rum-core/src/boot/preStartRum.ts

```diff
@@ -140,6 +141,8 @@ export function createPreStartStrategy(
         return
       }
 
+      callHook(initConfiguration.plugins, 'onInit', { initConfiguration, publicApi })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏To anticipate a bit future hooks, it could be nice to have
```suggestion
      registerePlugins(initConfiguration.plugins) // At some point we could have some checks like ensuring no duplicated plugins
      callHook('onInit', { initConfiguration, publicApi })
```

Reply:
What `registerPlugins` would do? I don't want to anticipate too much :) even `callHook` is a bit too much. Maybe I could remove this 'hook' concept and just move the code here:

```ts
initConfiguration.plugins?.forEach(plugin => plugin?.onInit({ initConfiguration, publicApi }))
```